codeframe/agents/agent_pool_manager.py

```
@@ -0,0 +1,365 @@
+            agent_type: Type of agent to create (backend, frontend, test)
+
+        Returns:
+            agent_id: ID of created agent
+
+        Raises:
+            ValueError: If unknown agent type
+            RuntimeError: If agent pool at maximum capacity
+        """
+        print(f"\n🏭 DEBUG: create_agent called with agent_type={agent_type}")
+        with self.lock:
+            print(f"🏭 DEBUG: Acquired lock")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)